arches_lingo/src/arches_lingo/App.vue

```diff
@@ -22,32 +27,36 @@ provide(userKey, { user, setUser });
 
 const router = useRouter();
 const route = useRoute();
+const toast = useToast();
+const { $gettext } = useGettext();
 
-const isAuthenticated = computed(() => {
-    return user.value && user.value.username !== "anonymous";
-});
+router.beforeEach(async (to, _from, next) => {
```

Comment:
This pattern no longer tacks on the `next` route for unpermissioned user login redirects ( eg `localhost:8000/login/search` )

Reply:
Much better solution, thanks.